apps/server/src/modules/fwu-learning-contents/uc/fwu-learning-contents.uc.spec.ts

```diff
@@ -0,0 +1,211 @@
+import { S3Client } from '@aws-sdk/client-s3';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { InternalServerErrorException, NotFoundException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { Logger } from '@src/core/logger';
+import { Readable } from 'stream';
+import { S3Config } from '../interface/config';
+import { FwuLearningContentsUc } from './fwu-learning-contents.uc';
+
+describe('FwuLearningContentsUC', () => {
+	const createParameter = () => {
+		const config = {
+			endpoint: '',
+			region: '',
+			bucket: 'test-bucket',
+			accessKeyId: '',
+			secretAccessKey: '',
+		};
+		const pathToFile = 'test/text.txt';
+		return { config, pathToFile };
+	};
+	let module: TestingModule;
+	let fwuLearningContentsUc: FwuLearningContentsUc;
+	let s3client: DeepMocked<S3Client>;
+
+	beforeAll(async () => {
+		const { config } = createParameter();
+
+		module = await Test.createTestingModule({
+			providers: [
+				FwuLearningContentsUc,
+				{
+					provide: 'FWU_S3_Client',
+					useValue: createMock<S3Client>(),
```

Comment:
Why do you do it as a provider but not use it in your UC?

Reply:
Changed client in UC to provider.